docs/integration/kubernetes.md

```diff
@@ -5,118 +5,96 @@ Kubernetes is a portable, extensible, open-source platform for managing containe
 
 ## Monitoring Kubernetes with Sematext
 
-Sematext Monitoring will give you detailed insights into your cluster’s health, performance metrics, resource counts amongst other important metrics. Speaking of metrics, check out [this ](https://sematext.com/docs/agents/sematext-agent/kubernetes/metrics/)page for a summarized list of the key metrics you can follow with Sematext as well as a short explanation for each one of them.
-
-
-
+Sematext Monitoring will provide you with detailed insights into your cluster's control plane components and their health, performance metrics, and resource counts, among other important metrics. Speaking of metrics, you can check out [this page ](https://sematext.com/docs/agents/sematext-agent/kubernetes/metrics/) for a summarized list of key metrics that you can track using Sematext. It also includes a short explanation for each metric.
 
 ![alt_text](https://sematext.com/wp-content/uploads/2020/03/Kubernetes-Pod-Overview.png "Sematext Monitoring")
 
+### Agent Install
+To start monitoring Kubernetes with Sematext install the Sematext Agent. Setting up the agent takes less than 5 minutes:
 
+1.  Create a new Infra App on https://apps.sematext.com/ui/monitoring-create by choosing the INFRA App card from the list of integrations.
+2.  Name your Infra App, select the Kubernetes environment and install the Sematext Agent based on your preferred installation method. Available options include kubectl and a Helm chart.
 
-### Helm Chart
-
-To start monitoring Kubernetes with Sematext install the Sematext Agent. The easiest way to do that is with a Helm chart. It’s available in the official charts repo and it will install to all nodes in your cluster. To install it run the following command:
-
-
-
-
-```
-helm install --name sematext-agent \
-  --set infraToken=<YOUR_INFRA_TOKEN> \
-  --set logsToken=<YOUR_LOGS_TOKEN> \
-  --set region=<"US" or "EU"> \
-  stable/sematext-agent
-```
-
-
-
-Check out [github](https://github.com/helm/charts/blob/master/stable/sematext-agent/README.md) for more details.
-
-
-### Sematext Operator
-
-You can also install Sematext Operator using this command:
-
-
-
-
-```
-kubectl apply -f https://raw.githubusercontent.com/sematext/sematext-operator/master/bundle.yaml
-```
-
-
-
-
-After the installation has finished you can create the SematextAgent resource that deploys the agent to all the nodes in your cluster.
-
-
+### Agent Configuration
+The Sematext Agent offers a versatile container engine monitoring and visibility solution that is easy to customize. For more information, please refer ot our [Agent Configuration for Kubernetes](https://sematext.com/docs/agents/sematext-agent/kubernetes/configuration/).
```

Comment:
Do you think it's worth to mention this doc too? https://github.com/sematext/docs/pull/501/files#diff-cc3391ebb402774b9fe7a68bdf5d4a6ddc072e4ceb44641863d389ec78461b25R1

Reply:
Yes. Maybe we can move ["How to prevent "Pending" status when installing Sematext Agent on a Kubernetes Cluster?"](https://github.com/sematext/docs/pull/501/files#diff-4d5030995c895c71c7872a167ca4b1f116a6bd5eb675e420139f00d115539b02) from App Monitoring to the newly created FAQ section here. WDYT?

Reply:
I'm not sure, looks like this docs/integration/kubernetes.md doc has a bit more product language.
Btw, I meant this one HostNetwork https://github.com/sematext/docs/pull/501/files#diff-cc3391ebb402774b9fe7a68bdf5d4a6ddc072e4ceb44641863d389ec78461b25R1
Not Pending FAQ. Pending should be OK because we have this line in your doc https://github.com/sematext/docs/pull/504/files/256623b85247a76c66631330d6eb258087b282d7#diff-382e8238bbfde042b768cae0edbd294f96fdf5019828d9ee32e596348dbe5a0eR873

Reply:
Thanks! Added `hostnetwork` section in FAQ section